powerauth-restful-security-spring-annotation/src/main/java/io/getlime/security/powerauth/rest/api/spring/encryption/EncryptorFactory.java

```diff
@@ -85,8 +85,8 @@ public PowerAuthNonPersonalizedEncryptor buildNonPersonalizedEncryptor(String ap
                     encryptionKeyResponse.getEphemeralPublicKey()
             );
         } catch (PowerAuthClientException ex) {
-            logger.warn(ex.getMessage(), ex);
-            throw new PowerAuthEncryptionException(ex.getMessage());
+            logger.warn("Encryption failed, error: {}", ex.getMessage());
+            throw new PowerAuthEncryptionException();
```

Comment:
Let's pass `ex` as a parameter to the new exception to keep the cause somewhere.